examples/multisource_adapt/README.md

```diff
@@ -0,0 +1,38 @@
+# Image Classification: Multi-source Domain Adaptation on Images (e.g. Office, Digits) with Lightning
+
+### 1. Description
+
+This example demonstrates multi-source domain adaptation methods with application in image object detection/recognition.
+
+### 2. Usage
+
+* Datasets:
+  * Digits (10 classes), 4 domains: MNIST, Modified MNIST (MNISTM), UPSP, SVHN
+  * Office-31 (31 classes), 3 domains: Amazon (A), DSLR (D), Webcam (W)
+  * Office-Caltech (10 classes), 4 domains: Amazon (A), DSLR (D), Webcam (W), Caltech (C)
+* Algorithms: M3SDA, MFSAN
+* Example: Caltech, DSLR, and Webcam (source) to Amazon (target) using M3SDA and MFSAN
+
+`python main.py --cfg configs/Office2A-M3SDA.yaml --gpus 0`
+
+`python main.py --cfg configs/Office2A-MFSAN.yaml --gpus 0`
+
+### 3. Related `kale` API
+
+`kale.embed.image_cnn`: Extract features from small-size (32x32) images using CNN.
```

Comment:
is office data still `small-size (32x32)`?

Reply:
No. This example also supports digits data.